superset/config.py

```diff
@@ -191,11 +191,16 @@ def _try_json_readsha(filepath, length):  # pylint: disable=unused-argument
 DRUID_TZ = tz.tzutc()
 DRUID_ANALYSIS_TYPES = ["cardinality"]
 
-# Legacy Druid connector
+# Legacy Druid NoSQL (native) connector
 # Druid supports a SQL interface in its newer versions.
 # Setting this flag to True enables the deprecated, API-based Druid
 # connector. This feature may be removed at a future date.
-DRUID_IS_ACTIVE = False
+DRUID_IS_ACTIVE = True
+
+# If Druid is active whether to include the links to scan/refresh Druid datasources.
+# This should be disable if you are trying to wean yourself off of the Druid NoSQL
+# connector.
+DRUID_METADATA_VIEW_LINKS = True
```

Comment:
maybe `DRUID_METADATA_LINKS_ENABLED`?

Reply:
Agreed. The `VIEW` term related to FAB views but the way it reads it's misleading.